modules/installation/pages/uyuni-server-setup.adoc

```diff
@@ -0,0 +1,176 @@
+Passwords must be at at least seven characters in length, and must not contain spaces, single or double quotation marks (``'`` or ``"``), exclamation marks (``!``), or dollar signs (``$``).
+Always store your passwords in a secure location.
++
+
+// image::quickstart-mgr-setup5-ncurses.png[scaledwidth=80%]
+
+. Click btn:[Next] to continue.
+
+. Click btn:[Yes] to run setup when prompted.
+
+. When setup is complete, click btn:[Next] to continue.
+You will see the address of the {productname} {webui}.
+
+. Click btn:[Finish] to complete {productname} setup.
+
+// In the next section you will create the administrator's account and synchronize with {scc}.
+
+
+== Create the Main Administration Account
+
+This section covers how to create your organization's main administration account for {productname}.
+
+[WARNING]
+====
+The main administration account has the highest authority within {productname}.
+Ensure you keep access information for this account secure.
+
+We recommend that you create lower level administration accounts for organizations and groups.
+Do not share the main administration access details.
+====
+
+
+.Procedure: Setting Up the Main Administration Account
+
+. In your web browser, enter the address for the {productname} {webui}.
+This address was provided after you completed setup.
+For more information, see xref:uyuni-server-setup.adoc#server-setup-yast[].
+
+. Log in to the {webui}, navigate to the menu:Create Organization[Organization Name] field, and enter your organization name.
+
+. In the menu:Create Organization[Desired Login] and menu:Create Organization[Desired Password] fields, enter your username and password.
+
+. Fill in the Account Information fields including an email for system notifications.
+
+. Click btn:[Create Organization] to finish creating your administration account.
++
+
+// image::quickstart-mgr-setup-admin1.png[scaledwidth=80%]
+
+When you have completed the {productname} {webui} setup, you are taken to the menu:Home[Overview] page.
+// In the next section you will prepare the server for connecting the first client.
+
+
+== Optional: Synchronizing Products from {scc}
+
+{scc} (SCC) maintains a collection of repositories which contain packages, software and updates for all supported enterprise client systems.
+These repositories are organized into channels each of which provide software specific to a distribution, release, and architecture.
+After synchronizing with SCC, clients can receive updates, be organized into groups, and assigned to specific product software channels.
+
+This section covers synchronizing with SCC from the {webui} and adding your first client channel.
+
+[NOTE]
+====
+For Uyuni, synchronizing products from {scc} is optional.
+====
+
+Before you can synchronize software repositories with SCC, you will need to enter organization credentials in {productname}.
+In previous versions, mirror credentials were used instead.
```

Comment:
I am afraid that despite the UI is confusing, we still use mirror  credentials:

![image](https://user-images.githubusercontent.com/4226070/69347464-c8a1ad80-0c74-11ea-87df-947d375d5d2e.png)

And I am pretty sure it's the same for SUSE Manager.

Reply:
ok, I remove this sentence.